examples/experimental/openvino/mobilenet_v2/main.py

```diff
@@ -11,27 +11,25 @@
  limitations under the License.
 """
 
-from typing import Iterable
-from typing import Any
 import subprocess
-from pathlib import Path
 import time
+from pathlib import Path
+from typing import Any
+from typing import Iterable
 
 import openvino.runtime as ov
 import torch
 import torchvision
-import nncf
 
+import nncf
```

Comment:
The `import nncf` should be in 2rd group of imports (related third-party imports).